.github/workflows/ci.yml

```diff
@@ -15,33 +15,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5, 2.6, 2.7, '3.0', 3.1, 3.2, 3.3, jruby-9.2, jruby-9.3, jruby-9.4]
-        gemfile: [rails-5.2.gemfile, rails-6.0.gemfile, rails-6.1.gemfile, rails-7.0.gemfile, rails-main.gemfile]
+        ruby: [2.7, '3.0', 3.1, 3.2, 3.3, jruby-9.2, jruby-9.3, jruby-9.4]
```

Comment:
JRuby 9.2 and 9.2 are < 2.7 IIRC. I think only 9.4 is needed here.

Reply:
Cool, I have no idea haha